packages/grafana-ui/src/components/ButtonCascader/_ButtonCascader.scss

```diff
@@ -1,20 +1,25 @@
 .rc-cascader {
   font-size: 12px;
 
+  &-dropdown {
+    position: absolute;
+    // Required, otherwise the portal that the popup is shown in will render under other components
+    z-index: 9999;
```

Comment:
this is just moved from https://github.com/grafana/grafana/pull/44241/files#diff-08368c9249935c21e12683080f8e5b516060b89adf4fd18771e3f63ed8d85a0aL216